src/devices/machine/upd765.cpp

```diff
@@ -376,6 +383,18 @@ READ8_MEMBER(upd765_family_device::msr_r)
 			//msr |= MSR_CB;
 		}
 
+	// log msr when changed
+	if (VERBOSE)
+	{
+		static u8 last_msr = 0;
```

Comment:
Oh no, this is a deal-breaker.  No non-const function statics.  It breaks save state support and systems with multiple instances of the same device.